src/app/_preview.scss

```diff
@@ -1,20 +1,24 @@
 @import "../styles/settings/colors";
 @import "../styles/mixins/respond";
-@import "../components/m-main-navigation/styles/stroke";
 
 $sg-patternlink-y-padding: 6px !default;
 $sg-patternlink-padding-side: 20px !default;
 $sg-patternlink-border-side-width: 1px !default;
 $sg-patternlink-border-side-color: $color-prim-gray-silver !default;
 $sg-patternlink-margin-side: 20px !default;
-
-body {
-  background: $color-prim-gray-alabaster;
-}
+$sg-main-title-size: 2.8rem;
 
 $sg-white: $color-prim-white;
 $bg-button: $color-axa-blue;
 
+$sg-black-syntaxhighlighter: #000;
+
+$sg-paddings-page: 3%;
```

Comment:
I don't know what is this for, but our guidlines define fixed width `px` gutters for the grid -> `15px`